library/core/class.locale.php

```diff
@@ -357,7 +357,9 @@ public function translate($code, $default = false) {
 
         $this->EventArguments['Code'] = $code;
```

Comment:
We should move event argument setting inside of the conditional as well

Reply:
Done.